frontend/src/components/form/CourtLocationSelect.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import _ from 'lodash'
```

Comment:
A comment that should be made here: `lodash` adds a fair bit of overheard. if you just need a single method, we can import just that method to reduce what's in our final client bundle pretty significantly. not a big deal as we're not that concerned with optimizing for mobile/bad connections etc but for clerks in remote areas, they only get 6MB download speeds which on some days can be pretty trying so every bit does help, at least imo (maybe you can tell I am subject to telus rural wifi itis)

```
import { times } from 'lodash'   // 68.81kB  :(
import times from 'lodash/times' //  2.08kB! :)
```

the above is just from some article that compares the size of each import.

Reply:
Thanks, good to know. 👍   I didn't notice `import times from 'lodash/times'` is much smaller.